res/lang/rooms/rooms_en.txt

```diff
@@ -82,10 +82,10 @@ ROOMAGENCY_ROOM_SUPERMARKET_IN_RANGE_TEXT1         = Convenient: just around the
 ROOMAGENCY_ROOM_SCRIPTAGENCY_IN_RANGE_TEXT1        = Just a short hop from the %X%.
 ROOMAGENCY_ROOM_OFFICE_IN_RANGE_TEXT1              = Nearby your own office!
 ROOMAGENCY_AVAILABLE_FOR_RENT_OF_X_PER_DAY         = Available for just |b|%X%/gameday|/b|.
-ROOMAGENCY_SIGNING_WILL_COST_A_SINGLE_PAYMENT_OF_X = Signing will cost a single payment of |b|%X%|/b|.
+ROOMAGENCY_SIGNING_WILL_COST_A_SINGLE_PAYMENT_OF_X = A one-time payment of |b|%X%|/b| is due upon signing.
 
-ROOMAGENCY_ROOM_CURRENTLY_NOT_AVAILABLE_BUT_THIS_MIGHT_CHANGE1 = This room is still rented, but with a nice advice the tenant could release the room.
-ROOMAGENCY_ROOM_CURRENTLY_NOT_AVAILABLE_BUT_THIS_MIGHT_CHANGE2 = This room is still rented, but with a little persuasion, the tenant might well be willed to cancel this room rental.
+ROOMAGENCY_ROOM_CURRENTLY_NOT_AVAILABLE_BUT_THIS_MIGHT_CHANGE1 = This room is still rented, but with a friendly suggestion the tenant might release the room.
```

Comment:
In that forum thread I meant it's more about the way and style how the message can be delivered so that it isn't missed, not so much about the content. But ok, a stronger hint here maybe could help too. When is _CHANGE1 vs _CHANGE2 displayed? Clicking more often?

Reply:
ROOMAGENCY_ROOM_CURRENTLY_NOT_AVAILABLE_BUT_THIS_MIGHT_CHANGE
-> this is the text to use


ROOMAGENCY_ROOM_CURRENTLY_NOT_AVAILABLE_BUT_THIS_MIGHT_CHANGE1
ROOMAGENCY_ROOM_CURRENTLY_NOT_AVAILABLE_BUT_THIS_MIGHT_CHANGE2
ROOMAGENCY_ROOM_CURRENTLY_NOT_AVAILABLE_BUT_THIS_MIGHT_CHANGE3
ROOMAGENCY_ROOM_CURRENTLY_NOT_AVAILABLE_BUT_THIS_MIGHT_CHANGE4
ROOMAGENCY_ROOM_CURRENTLY_NOT_AVAILABLE_BUT_THIS_MIGHT_CHANGE5
-> Each time you request a locale with the key ROOMAGENCY_ROOM_CURRENTLY_NOT_AVAILABLE_BUT_THIS_MIGHT_CHANGE one of the 5 entries will be chosen
-> used in dialogues, tooltips ... to allow variation

Reply:
I think, I understood what you meant in the forum thread. And I'd say, players generally won't read the manual and won't press F1 for help. But they probably will click on the room entry and will see this text (or more precisely, one random version).

So, basically additional versions could be added as well ("Probably nothing short of an explosion will make the current tenant leave.")